.github/workflows/openbsd.yml

```diff
@@ -0,0 +1,48 @@
+name: C CI on OpenBSD
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: OpenBSD-${{ matrix.config.name }}
+    runs-on: macos-12
+    env:
+      CFLAGS: ${{ matrix.config.cflags }}
+      NROFF: ${{ matrix.config.nroff }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+         - {
+           name: default64,
+           nroff: nroff,
+           cflags: ""
+         }
+         - {
+           name: no_obsolete64,
+           nroff: true,
```

Comment:
I think this is yaml and needs quoting here, else it's a bool

Reply:
If that's the case, we need it fixed across all the GitHub builders.

Reply:
At least it works, because the build runs "true -man something.8".

Reply:
It probably works because the stringification turns it into "true".

Good thing we are not testing "yes" instead. ;)

Reply:
Works quoted as well, so I quote it in the new file.